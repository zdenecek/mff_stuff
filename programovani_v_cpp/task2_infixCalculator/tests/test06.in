6
a=1
$a
b=3.5
$a+$b+$a/$a
=23
$
