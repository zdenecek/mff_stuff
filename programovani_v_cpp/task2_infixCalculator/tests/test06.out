1
1
3.5
5.5
23
23
